pwd_guess.py

```diff
@@ -930,11 +950,16 @@ def train_model(self, serializer):
         max_accuracy = 0
         if self.config.scheduled_sampling:
             self.ctable.init_model(self.model)
+            if self.config.tensorboard:
+            self.callback.set_model(self.model)
```

Comment:
This line looks suspicious. Did you mean to indent it so that it is guarded by the if statement? 

BTW, the pylint tool will catch this sort of thing if you run it before commits. Also the unittests fail because of it.

Reply:
Also, should probably in general move the set_model call so that it is outside the check for scheduled_sampling.